src/main/webapp/view.js

```diff
@@ -15,22 +15,53 @@
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
 
+/** Timestamp that video, disccussion, and transcript were synced to. */
+let lastSyncedTime;
+
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
 initialize();
 
 /**
- * Initializes the video player and discussion section
+ * Initializes the timer, video player, and discussion section
  * for the lecture view page.
  */
 async function initialize() {
   window.loadVideoApi();
-  window.intializeDiscussion();
+  window.loadDiscussion();
+  // TODO: Adjust time interval.
+  window.timer = window.setInterval(syncEverythingToVideo, 1000);
```

Comment:
Since this function does so little, what do you think about inlining it?
```suggestion
  window.timer = window.setInterval(() => {
    sync(window.videoPlayer.getCurrentTime(), false);
  }, 1000);
```